app/components/workbench/Workbench.client.tsx

```diff
@@ -116,6 +117,21 @@ export const Workbench = memo(({ chatStarted, isStreaming }: WorkspaceProps) =>
     }
   }, []);
 
+  const handleLoadFromFileSystem = useCallback(async () => {
+    setIsLoading(true);
+
+    try {
+      const directoryHandle = await window.showDirectoryPicker();
+      await workbenchStore.loadFromFileSystem(directoryHandle);
```

Comment:
I'm really glad to see that this is already part of the workbench store, that's one of the major things I have been wanting to track down as free time allows me to. This call existing is important. 👍